src/main/java/com/faforever/client/filter/ChatUserFilterController.java

```diff
@@ -0,0 +1,115 @@
+package com.faforever.client.filter;
+
+import com.faforever.client.chat.ChatChannelUser;
+import com.faforever.client.chat.ListItem;
+import com.faforever.client.domain.LeaderboardBean;
+import com.faforever.client.domain.PlayerBean;
+import com.faforever.client.game.PlayerStatus;
+import com.faforever.client.i18n.I18n;
+import com.faforever.client.leaderboard.LeaderboardService;
+import com.faforever.client.player.Country;
+import com.faforever.client.player.CountryFlagService;
+import com.faforever.client.theme.UiService;
+import com.faforever.client.util.RatingUtil;
+import javafx.util.StringConverter;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.Optional;
+
+@Component
+@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+public class ChatUserFilterController extends AbstractFilterController<ListItem> {
+
+  private final CountryFlagService countryFlagService;
+  private final LeaderboardService leaderboardService;
+
+  private final static int MIN_RATING = -1000;
+  private final static int MAX_RATING = 4000;
+
```

Comment:
We should probably standardize these across the replay filter and here

Reply:
Sorry, I do not understand what you mean by `replay filter`?

Reply:
https://github.com/FAForever/downlords-faf-client/blob/b79be72f1852f8c0da635d3cd57e49acc67adbd7/src/main/java/com/faforever/client/replay/OnlineReplayVaultController.java#L144-L146
Do you want to replace these values?

Reply:
yes just for consistency in terms of what we consider the max and min rating